src/sanescansrv/server.py

```diff
@@ -505,7 +510,7 @@ async def scan_status_get() -> AsyncIterator[str] | tuple[AsyncIterator[str], in
 
     if status == ScanStatus.DONE:
         filename = data[0]
-        return app.redirect(f"/{filename}")
+        return app.redirect(f"/scans/{filename}")
```

Comment:
Thinking a bit more about security, I think it might be better if we register a new subpage, ex with 
```python
@app.get("/scans/<scan_filename>")
async def handle_scans_get(scan_filename: str) -> None:
    ...
```
and then use whatever quart's stream file function is (forgetting at time of writing) to send filename from tmp folder instead of making a symlink to tmp folder.

Reply:
I believe its `send_file`, I can do that and update the PR.